recipes/ffmpeg/all/conanfile.py

```diff
@@ -412,6 +414,9 @@ def build_requirements(self):
             self.win_bash = True
             if not self.conf.get("tools.microsoft.bash:path", check_type=str):
                 self.tool_requires("msys2/cci.latest")
+            if self.settings.arch == "armv8":
+                self.tool_requires("strawberryperl/5.32.1.1")
+                self.tool_requires("gas-preprocessor/0.0.0.cci20250710")
```

Comment:
```suggestion
            if self.settings.arch == "armv8" and is_msvc(self):
                self.tool_requires("gas-preprocessor/[*]")
```

- strawberryperl may not be needed because `msys2` already includes `perl.exe`  
- unconstrained version range, I would expect this to be updated very infrequently and also to remain compatible
- I think `gas-preprocessor` is only needed to be able to use the msvc assembler - so it wouldn't be needed only because "on windows"